cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -58,7 +58,16 @@
         <div class="o-video-player_video-container
                     show-on_video-playing
                     {{ 'o-video-player_video-container__flexible' if is_flexible else '' }}">
-            <div class="o-video-player_iframe-container"></div>
+            <div class="o-video-player_iframe-container">
+                <iframe class="o-video-player_iframe"
+                        width="100%"
+                        src="{{ video_url }}"
+                        frameborder="0"
+                        allow="autoplay"
+                        allowfullscreen="true"
+                        scrolling="no">
+                </iframe>
+            </div>
             <div class="o-video-player_video-actions-container">
                 <div class="o-video-player_video-actions">
                     <button class="a-btn o-video-player_close-btn" href="/">
```

Comment:
Another thing that's out of scope for this PR: I was testing with a screen reader to make sure that it didn't read both the fallback `a` play button and the JS-enabled `button` play button. It doesn't, but unfortunately, I discovered that I must've not tested this updated FCM thoroughly in the first place. Definitely some weirdness happening. But let's not let that hold up this work.